notario/tests/test_functional.py

```diff
@@ -61,15 +61,16 @@ def test_most_simple_validation(self):
         assert  "-> a -> a did not match 'b'" in error
 
     def test_required_key_is_missing(self):
-        data = {'a': 'a', 'c': 'c'}
-        schema = (('a', 'a'), ('b', 'b'), ('c', 'c'))
+        data = {'k1': 'v1', 'k3': 'v3'}
```

Comment:
this explains so very well how I missed out on the bug, because you are identifying the keys and values, and making sure that the key is reported, the previous implementation was using the value. Thanks for digging into this @dmick